app/models/subscription.rb

```diff
@@ -63,7 +63,29 @@ def pending_start_date
     (created_at.end_of_month + 1.day).to_date
   end
 
+  def next_pending_start_date
+    following_subscription = next_subscription
+
+    return unless following_subscription
+    return unless following_subscription.pending?
+
+    last_interval_date(self)
+  end
+
   def already_billed?
     fees.subscription_kind.any?
   end
+
+  def last_interval_date(subscription)
```

Comment:
Do we need to pass, the subscription, can't we use the `self` one instead?

Reply:
At first I planned to use this method twice (for self and for next subscription) but since I'm just using it on self for now, changed it :)